a '\n' are executing instead readit causing SC2034 ERROR

Correct Code (Passed in ShellCheck)
`MyBalR=$(echo "$MyBal" | awk '{printf("%d\n",$1 + 0.5)}')`

Get this on action-sh-checker v0.1.5 (and4 too) (are doing the \n on checking instead read it)
````
MyBalR=$(echo "$MyBal" | awk '{printf("%d
",$1 + 0.5)}')
^----^ SC2034: MyBalR appears unused. Verify use (or export if used externally).
````

The error: SC2034 is not related with this break line "\n".

When shellcheck run over your script, it identifies the problem SC2034, and when sh-checker shows the output the character \n is converted to a new line.

--- a/entrypoint.sh
+++ b/entrypoint.sh
@@ -84,8 +84,8 @@ exit_code=0
 if [ "$SHELLCHECK_DISABLE" != "1" ]; then
 	echo -e "Validating shell scripts files using shellcheck\n"
 	# shellcheck disable=SC2086
-	shellcheck_error="$(shellcheck $sh_files)"
-	shellcheck_code="$?"
+	shellcheck_error=$( (shellcheck $sh_files) | while read -r x; do echo "$x"; done; )
+	shellcheck_code="${PIPESTATUS[0]}"
 fi
 
 if [ "$SHFMT_DISABLE" != "1" ]; then

